src/exception/repo/program.ts

```diff
@@ -16,86 +16,69 @@
  * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-import { isArray } from 'lodash';
 import mongoose from 'mongoose';
-import { loggerFor } from '../logger';
-import { ExceptionValue, ObjectValues, ProgramException } from './types';
+import { loggerFor } from '../../logger';
+import { DatabaseError } from '../error-handling';
+import { ExceptionValue, ProgramException } from '../types';
 
 const L = loggerFor(__filename);
 
 const programExceptionSchema = new mongoose.Schema<ProgramException>({
   programId: String,
   exceptions: [
     {
+      program_name: String,
       schema: String,
-      coreField: String,
-      exceptionValue: { type: String, enum: Object.values(ExceptionValue) },
+      requested_core_field: String,
+      requested_exception_value: { type: String, enum: Object.values(ExceptionValue) },
```

Comment:
Not a request for change, but I'm curious about the change from camelCase.
Care to elaborate?

Reply:
TLDR: keeping as snake_case throughout reduced complexity and is consistent with code already present
two main things.
- our clinical entity records get saved as snakecase eg. `number_lymph_nodes_positive` so there is prior work that is snakecased
- tsv upload is snakecased, so there was a lot of additional code just to convert property names back and forth for the JS idiomatic camelCase.

Reply:
You're right, we have to keep it consistently inconsistent.
Unfortunate, but not in the scope of this PR, as you well point out.